lucene/highlighter/src/java/org/apache/lucene/search/uhighlight/LengthGoalBreakIterator.java

```diff
@@ -60,12 +83,15 @@ private LengthGoalBreakIterator(BreakIterator baseIter, int lengthGoal, boolean
   @Override
   public String toString() {
     String goalDesc = isMinimumLength ? "minLen" : "targetLen";
-    return getClass().getSimpleName() + "{" + goalDesc + "=" + lengthGoal + ", baseIter=" + baseIter + "}";
+    return getClass().getSimpleName() + "{" + goalDesc + "=" + lengthGoal + ", fragAlign=" + fragmentAlignment +
+        ", baseIter=" + baseIter + ", currentCache=" + currentCache + "}";
```

Comment:
I'm going to remove currentCache from the toString when I commit because I don't think it belongs.  It's an internal detail.